installer/kfserving/description.yaml

```diff
@@ -1,7 +1,7 @@
 name: kfserving
 description: |
   Serverless Inferencing on Kubernetes
-namespace: kfserving-system
+namespace:
```

Comment:
Does the field have to be present, even if it doesn't contain a value ?

Reply:
It does not